docs/source/tutorials-basic/driving.rst

```diff
@@ -0,0 +1,197 @@
+.. _tutorials-basic-driving-chapter:
+
+**************************
+Driving Around
+**************************
+
+Driving the GoPiGo3 around is only a matter of choosing the right methods from the :py:class:`~easygopigo3.EasyGoPiGo3` class.
+In all of our examples, the :py:mod:`~easygopigo3` is required and needs to be imported and a :py:class:`~easygopigo3.EasyGoPiGo3` object
+has to be instantiated this way:
+
+.. code-block:: python
+
+   from easygopigo3 import EasyGoPiGo3 # importing the EasyGoPiGo3 class
+   gpg = EasyGoPiGo3() # instantiating a EasyGoPiGo3 object
+
+Instantiating :py:class:`~easygopigo3.EasyGoPiGo3` object can fail if the GoPiGo3 is not present.
```

Comment:
if the GoPiGo3 is not present or if it has the wrong version of the firmware.  Or is that an exception being raised?

Reply:
Up there I'm referencing the constructor of `EasyGoPiGo3` and it specifically says to look up for the possible exceptions that can get thrown by it. To answer your question, yes an exception is thrown.

Reply:
but it still fails if the GPG has the wrong firmware.